README.md

```diff
@@ -390,13 +390,21 @@ php artisan discord:interactions:register
 
 #### Testing Webhook Locally
 
-For local development, use tools like ngrok to expose your local server:
+For local development, we recommend using [expose](https://expose.dev/) to expose your local server. Expose is a popular choice among Laravel/PHP developers for creating secure tunnels to localhost.
+
+First, install expose globally:
+
+```shell
+composer global require exposedev/expose
+```
+
```

Comment:
Added an explanation on the need to create an expose account and set a token.
```
expose token YOUR_TOKEN
```
https://expose.dev/docs/getting-started/authentication